src/services/GooglePlacesService.ts

```diff
@@ -1,8 +1,9 @@
 import axios, { AxiosError, AxiosResponse } from 'axios';
 import * as dotenv from 'dotenv';
-import { RestaurantDetails } from '.././types/RestaurantDetails';
+import { RestaurantDetails } from '../types/RestaurantDetails';
 import { RestaurantInfo } from '../types/RestaurantInfo';
 
+/* eslint-disable  @typescript-eslint/no-explicit-any */
```

Comment:
Thanks for adding these, I just created an issue to remind myself to handle these later: https://github.com/ubclaunchpad/PromoPal-frontend/issues/2